test/ogf.jl

```diff
@@ -1,23 +1,23 @@
 @testset "test ogf" begin
     @testset "test nlp ogf" begin
-        @testset "gaslib 40 nlp ogf" begin
+        @testset "case 6 ogf" begin
```

Comment:
why did the objective value change on these problems?

Reply:
I fixed some bugs in the per unit conversions and changed certain values in the data files to match old results.